airbyte-cdk/bulk/toolkits/load-object-storage/build.gradle

```diff
@@ -16,3 +16,12 @@ dependencies {
     testFixturesImplementation testFixtures(project(":airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-csv"))
     testFixturesImplementation testFixtures(project(":airbyte-cdk:bulk:toolkits:bulk-cdk-toolkit-load-parquet"))
 }
+
+project.tasks.matching {
```

Comment:
nit: The indentation looks off. Should this have run through the pre-commit hook?
Not sure if we are still trying to enforce this

Reply:
it passes the formatter?